.github/workflows/stale_marker.yml

```diff
@@ -12,8 +12,8 @@ jobs:
           stale-issue-message: "This issue is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 7 days."
           stale-pr-message: "This PR is stale because it has been open 90 days with no activity."
           close-issue-message: "This issue was closed because it has been stalled for 30 days with no activity."
-          days-before-issue-stale: 90
-          days-before-pr-stale: 90
+          close-pr-message: "This PR was closed because it has been stalled for 180 days with no activity."
+          days-before-stale: 90
           days-before-issue-close: 30
```

Comment:
is this 90 + 30 (stale + close) ?

Reply:
Yes. So if issue has not activity for 90 days, it is marked as stale. After that 30 days are counted and issue is closed in case of lack of activity, since adding stale label actually is sort of activity itself.